docs/data/base/components/accordion/accordion.md

```diff
@@ -0,0 +1,15 @@
+---
+product: base
```

Comment:
Since [this PR](https://github.com/mui/material-ui/pull/37593) has merged, you'll have to use `base-ui` instead of `base`

Reply:
@brijeshb42 Do I need to update all of them now? Or can I do it on a separate PR?

Reply:
Asking as I thought only these new pages I added were outdated but realized all of them are using `base` instead of `base-ui`, so not sure if doing it right now is the best moment. No worries about doing it though ⎯ just making sure!

Reply:
Have you rebased your changes with latest master? All md files are now using `base-ui` only.

Reply:
Ah perfect ⎯ just did that. Thanks for the heads-up 🤙